test/Microsoft.ML.Tests/Scenarios/OvaTest.cs

```diff
@@ -125,9 +128,9 @@ public void OvaLinearSvm()
                             new TextLoader.Column("Features", DataKind.Single, new [] { new TextLoader.Range(1, 4) }),
                         }
             });
-
             // Data
-            var data = mlContext.Data.Cache(reader.Load(GetDataPath(dataPath)));
+            var textData = reader.Load(GetDataPath(dataPath));
+            var data = mlContext.Data.Cache(mlContext.Transforms.Conversion.MapValueToKey("Label").Fit(textData).Transform(textData));
```

Comment:
>var data  [](start = 12, length = 9)

Maybe for lines like this in this file, break the steps out a bit for easier reading?